tests/test_serializers.py

```diff
@@ -29,9 +33,14 @@ def setUp(self):
             'password': self.password
         }
 
+    def get_serializer(self, **kwargs):
+        serializer = JSONWebTokenSerializer(**kwargs)
+        serializer.context['request'] = Request(HttpRequest()),
```

Comment:
Is the `,` at the end of this line a mistake? That creates a tuple for the value of `serializer.context['request']` which doesn't seem quite right